shared/actions/provision.tsx

```diff
@@ -213,21 +226,25 @@ class ProvisioningManager {
       throw new Error('Tried to submit a code but missing in store')
     }
 
-    response.result({code: null, phrase: state.provision.codePageOutgoingTextCode.stringValue()})
+    response.result({phrase: state.provision.codePageOutgoingTextCode.stringValue(), secret: null as any})
```

Comment:
wants Bytes but i was afraid of sending Buffer(0) so sending null as it was before